src/ert/config/analysis_module.py

```diff
@@ -99,15 +99,15 @@ class IESSettings(BaseSettings):
     ] = "subspace_exact"
     ies_max_steplength: Annotated[
         float,
-        Field(ge=0.1, le=1.0, title="Gauss–Newton maximum steplength"),
+        Field(ge=0.1, le=1.0, title="Gauss-Newton maximum steplength"),
```

Comment:
Changing en-dashes to simple dashes? en-dash is the correct way of concatenating scientists names like this (says a LaTeX-fanboy)